packages/@okta/vuepress-site/guides/sign-into-web-app/require-authentication/aspnetcore/reqautheverything.md

```diff
@@ -13,14 +13,14 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-Also, you can opt to create a `BaseProtectedController` decorated with the `Authorize` attribute and make all of your controllers inherit it:
+Or, you could create a `BaseProtectedController`and make all of your controllers to inherit from it:
```

Comment:
Or, you could create a `BaseProtectedController`and make all of your controllers inherit from it: